docs/pages/getting-started.html

```diff
@@ -52,6 +52,12 @@ <h3 class="usa-heading">For developers</h3>
 
 <p>The standards are designed to support both older and newer browsers through progressive enhancement, and officially support Internet Explorer 9 and up, and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage today, and most agency websites should be able to safely begin support at Internet Explorer 9.</p>
 
+<h4>Progressive enhancement:</h4>
+
+<p>Progressive enhancement is a web design and development strategy emphasizing accessibility through <a href="http://www.w3.org/html/wg/wiki/Guide/Syntax-Semantics" target="_blank" title="World Wide Web Consortium - Guide/Syntax and Semantics">semantic HTML markup</a>. Semantic HTML allows older browsers to present the content of a document to users. The semantic markup is layered with external technologies to enhance the user experience (specifically, CSS and JavaScript), without necessarily effecting the underlying funcitonality of the document and site.</p>
```

Comment:
Hey! A few notes here:

I'd reword this:

layered with external technologies to enhance the user experience (specifically, CSS and JavaScript)

to this:

layered with external technologies (specifically, CSS and JavaScript) to enhance the user experience 

(Does that make sense/is that still correct?)

Also, please change `effecting` to `affecting`